File: base-ui/src/app/models/shipping-address.ts
export class ShippingAddress {
    public identifier : string;
    public line1 : string;
    public line2 : string;
    public phone : number;
    public city : string;
    public zone : string;
    public username : string;
    
    constructor(init? : Partial<ShippingAddress>){
        Object.assign(this, init);
    }
    

    set Identifier(value) {
        this.identifier = value;
    }
    set Line1(value) {
        this.line1 = value;
    }
    set Line2(value) {
        this.line2 = value;
    }
    set Phone(value) {
        this.phone = value;
    }
    set Username(value) {
        this.username = value;
    }
    set City(value) {
        this.city = value;
    }
    set Zone(value) {
        this.zone = value;
    }
    get Identifier() {
        return this.identifier;
    }
    get Line1() {
        return this.line1;
    }
    get Line2() {
        return this.line2;
    }
    get Phone() {
        return this.phone;
    }
    get Username() {
        return this.username;
    }
    get City() {
        return this.city;
    }
    get Zone() {
        return this.zone;
    }
}

File: Base/Base/Models/ShippingAddress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Base
{
    public class ShippingAddress
    {
        public string Identifier { get; set; }
        public string Line1 { get; set; }
        public string Line2 { get; set; }
        public int Phone { get; set; }
        public string City { get; set; }
        public string Zone { get; set; }
        public string Username { get; set; }

        public void Show()
        {
            Console.WriteLine("Identifier: " + Identifier);
            Console.WriteLine("Line 1: " + Line1);
            Console.WriteLine("Line 2: " + Line2);
            Console.WriteLine("Phone: " + Phone);
            Console.WriteLine("City: " + City);
            Console.WriteLine("Zone: " + Zone);
            Console.WriteLine("Username: " + Zone);
        }

        public static implicit operator List<object>(ShippingAddress v)
        {
            throw new NotImplementedException();
        }
    }
}
